File: react-js/Micro-frontend para uma empresa promotora de crédito/packages/core/src/plataforma-cliente/anexo/anexo.model.ts
import { DocumentTypeCode } from './document-types.enum';

export interface AnexoCriacaoModel {
  idTipoDocumento: DocumentTypeCode;
  anexoBase64: string;
  extensao: string;
}

export interface Anexo {
  linkAnexo: string;
  id: number;
  tipoDocumento: {
    id: DocumentTypeCode;
    codigo: string;
    nome: string;
  };
  dataCadastro: string;
  extensao: string;
}

File: net-core-apps/Monolito para uma promotora de crédito/Aplicacao/Model/Anexo/AnexoModel.cs
using System;

namespace Aplicacao
{
    public class AnexoModel
    {
        public int Id { get; set; }

        public string LinkAnexo { get; set; }

        public TipoDocumentoModel TipoDocumento { get; set; }

        public UsuarioModel Usuario { get; set; }

        public int IdUsuario { get; set; }

        public string AnexoBase64 { get; set; }

        public string Extensao { get; set; }

        public DateTime DataCadastro { get; set; }
    }
}
